core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerPool.scala

```diff
@@ -305,6 +308,13 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
     case RescheduleJob =>
       freePool = freePool - sender()
       busyPool = busyPool - sender()
+    case message: ByteSizeMessage =>
+      logging.info(
+        this,
+        s"user memory is reconfigured from ${latestUserMemory.toString} to ${message.userMemory.toString}")
+      latestUserMemory = message.userMemory
```

Comment:
Even if controllers check the validity of this value, it would be great to cross-check the value again in the invoker layer.

And regarding the check, I feel the application should not concern the condition of the underlying host.
But on the other hand, I am not sure it would be reasonable to allow a bigger memory than the invoker host has.

I would defer this to other reviewers.